core/package.json

```diff
@@ -3,7 +3,6 @@
   "module": "./dist/index.esm.js",
   "types": "./dist/index.d.ts",
   "exports": {
-    "react-server": "./dist/react-server.mjs",
```

Comment:
Will this cause issues when using inside Next.js? How will Webpack handle it on the server layer?

Reply:
This file can be removed, the `core/package.json` is root package.json now. webpack will follow the standard to read the `exports` in `package.json`. Removed in [5b9c0fc](https://github.com/vercel/swr/pull/2865/commits/5b9c0fc247f668db41eeecff5297cc4a076132fa)